python/ray/rllib/agents/impala/vtrace_policy_graph.py

```diff
@@ -282,6 +300,13 @@ def postprocess_trajectory(self,
                                other_agent_batches=None,
                                episode=None):
         del sample_batch.data["new_obs"]  # not used, so save some bandwidth
+        if self.config["rnd"]:
+            # augment batch with normalized observations for training rnd network
+            sample_batch["norm_obs"] = self.rnd_obs_filter(sample_batch["obs"])
```

Comment:
what if you just use tf.nn.batch_normalization?

Reply:
this way you can avoid using the obs filter twice. (or however many times, because you may actually need to sync the obs filter amongst workers)